packages/yourloops/pages/hcp/patients-list.tsx

```diff
@@ -256,6 +255,9 @@ class PatientListPage extends React.Component<RouteComponentProps, PatientListPa
         case "pending":
           return false; // TODO
         default:
+          if ((patient.teams?.includes(filterType) ?? false) === false) {
```

Comment:
```suggestion
          if ((!patient.teams?.includes(filterType)) {
```

Reply:
My next PR will change this code

Reply:
Fix in https://github.com/mdblp/blip/pull/400